package.json

```diff
@@ -39,7 +38,7 @@
     "build.watch": "lerna run --stream build.watch",
     "test": "lerna run --stream test",
     "test.watch": "lerna run --stream test.watch",
-    "ci-load": "node ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --iterations 10000",
+    "ci-load": "node ./packages/test/lib/load/all-in-one.js --server-address $TEMPORAL_TESTING_SERVER_URL --iterations 10000 --otel-url=''",
```

Comment:
Why is the otel-url empty? Do we not want to export anything?

Reply:
It's default curing CI because there's nowhere to look at what's exported.